app/controllers/katello/api/v2/capsule_content_controller.rb

```diff
@@ -54,6 +54,23 @@ def sync
       respond_for_async :resource => task
     end
 
+    api :GET, '/capsules/:id/content/sync',  'Synchronize the content to the capsule'
+    param :id, Integer, :desc => 'Id of the capsule', :required => true
+    def sync_status
+      @capsule_content = capsule_content
+    end
+
+    api :DELETE, '/capsules/:id/content/sync',  'Cancel running capsule synchronization.'
+    param :id, Integer, :desc => 'Id of the capsule', :required => true
+    def cancel_sync
+      tasks = capsule_content.cancel_sync
+      if tasks.empty?
+        render_message _('There\'s no running synchronization for this capsule.')
+      else
+        render_message _('Trying to cancel the synchronization...')
+      end
+    end
```

Comment:
I don't believe so, as long as the cancel_sync function still works.  I will add a user story to ensure that.